copilot/fastapi/app/kubernetes_api.py

```
@@ -0,0 +1,110 @@
+        
+    Returns:
+        dict: The JSON response from the Kubernetes API.
+        
+    Raises:
+        HTTPException: If the Kubernetes API is unavailable or returns an error.
+    """
+    settings = get_settings()
+    
+    # Determine headers
+    try:
+        with open(settings.kubernetes_token_path, "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)